nvflare/private/fed/client/client_engine_internal_spec.py

```diff
@@ -49,17 +49,21 @@ def get_client_name(self) -> str:
         pass
 
     @abstractmethod
-    def deploy_app(self, app_name: str, job_id: str, client_name: str, app_data) -> str:
-        """Deploys the app to specified run.
+    def deploy_app(self, app_name: str, job_id: str, job_meta: dict, client_name: str, app_data) -> str:
+        """Deploy the app to specified run.
```

Comment:
Google style docstring suggest "Deploys", let's just be consistent with other docstrings

Reply:
Will address this in another PR